test/components/node-problem-detector/nodeproblemdetector_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws aws_edge packet aks
```

Comment:
Can we also test on packet_arm? Will it work?

Reply:
let me try.

Reply:
I don't think the image is `packet_arm` compatible. Pipeline failed https://yard.lokomotive-k8s.net/builds/48502

Reply:
True, thanks for checking :) https://github.com/kubernetes/node-problem-detector/issues/135